swagger.yaml

```diff
@@ -129,6 +129,108 @@ paths:
         description: "`Chart` token"
         required: true
         type: string
+  "/{account}/spaces/{space}/datasets":
+    x-summary: Datasets
+    get:
+      operationId: listDatasetsInSpace
+      summary: List datasets for a space
+      description: Returns all `Datasets`s in a `Space`
+      responses:
+        '401':
+          description: Unauthorized
+          schema:
+            "$ref": "#/definitions/Unauthorized"
+        '404':
+          description: "`Dataset` not found"
+          schema:
+            "$ref": "#/definitions/NotFound"
+        '200':
+          description: "`Dataset` collection response"
+          schema:
+            "$ref": "#/definitions/Datasets"
+      parameters:
+      - name: account
+        in: path
+        description: "`Account` (`Workspace` or `User`) username"
+        required: true
+        type: string
+      - name: space
+        in: path
+        description: "`Space` token"
+        required: true
+        type: string
+      - name: filter
+        in: query
+        description: 'Returns a filtered list of `Dataset`s in a `Space` filtered
+          by whether their `created_at` or `updated_at` timestamps are `gt` (greater
+          than) or `lt` (less than) the datetime passed in (ISO8601 format). Example:
+          `?filter=created_at.gt.2019-10-23T06:23:01Z` returns all `Datasets` in a
+          space created after that date.'
+        required: false
+        type: string
+      - name: order
+        in: query
+        description: 'With the `order_by` param, returns all `Dataset`s in a `Space`
+          ordered in `asc` or `desc` order. Example: `?order=asc&order_by=created_at`'
+        required: false
+        type: string
+      - name: order_by
+        in: query
+        description: 'With the `order` param, returns all `Dataset`s in a `Space`
```

Comment:
```suggestion
        description: 'With the `order_by` param, returns all `Datasets` in a `Space`
```

Reply:
Thanks for catching this!  The same issue existed for reports.
<img width="728" alt="image" src="https://user-images.githubusercontent.com/1031483/207392263-d42fe7a2-7dd8-4106-90a5-2effc4207db6.png">